src/test/java/org/folio/rest/impl/EholdingsTitlesTest.java

```diff
@@ -0,0 +1,150 @@
+        .willReturn(new ResponseDefinitionBuilder()
+          .withBody(TestUtil.readFile(stubResponseFile))));
+
+    RestAssured.given()
+      .spec(spec).port(port)
+      .header(new Header(RestConstants.OKAPI_URL_HEADER, wiremockUrl))
+      .header(TENANT_HEADER)
+      .header(TOKEN_HEADER)
+      .when()
+      .get("eholdings/titles?page=1&filter[name]=Mind&sort=name")
+      .then()
+      .statusCode(200)
+
+      .body("meta.totalResults", equalTo(8766))
+      .body("data[0].type", equalTo("titles"))
+      .body("data[0].id", equalTo("1175655"))
+      .body("data[0].attributes.name", equalTo("The $1 Million Reason to Change Your Mind"))
+      .body("data[0].attributes.publisherName", isEmptyOrNullString())
+      .body("data[0].attributes.isTitleCustom", equalTo(false))
+      .body("data[0].attributes.subjects[0].type", equalTo("BISAC"))
+      .body("data[0].attributes.subjects[0].subject", equalTo("BUSINESS & ECONOMICS / Small Business"))
+
+      .body("data[0].attributes.identifiers[0].id", equalTo("7209484"))
+      .body("data[0].attributes.identifiers[1].id", equalTo("978-1-74216-894-4"))
+      .body("data[0].attributes.identifiers[2].id", equalTo("978-0-7303-7792-4"))
+
+      .body("data[0].attributes.identifiers[0].type", equalTo("ISSN"))
+      .body("data[0].attributes.identifiers[1].type", equalTo("ISBN"))
+      .body("data[0].attributes.identifiers[2].type", equalTo("ISBN"))
+
+      .body("data[0].attributes.identifiers[0].subtype", equalTo("Print"))
+      .body("data[0].attributes.identifiers[1].subtype", equalTo("Print"))
+      .body("data[0].attributes.identifiers[2].subtype", equalTo("Online"))
+
+      .body("data[0].attributes.publicationType", equalTo("Book"))
+
+      .body("data[0].relationships.resources.meta.included", equalTo(false));
+  }
+
+  @Test
+  public void shouldReturn400IfCountOutOfRange() {
+    RestAssured.given()
+      .spec(spec).port(port)
+      .header(TENANT_HEADER)
+      .header(TOKEN_HEADER)
+      .when()
+      .get("eholdings/titles?count=1000&page=1&filter[name]=Mind&sort=name")
+      .then()
+      .statusCode(400);
+  }
+
+  @Test
+  public void shouldReturn500WhenRMApiReturns500Error() throws IOException, URISyntaxException {
+    String wiremockUrl = host + ":" + userMockServer.port();
+    TestUtil.mockConfiguration("responses/configuration/get-configuration.json", wiremockUrl);
+    WireMock.stubFor(
+      WireMock.get(new UrlPathPattern(new RegexPattern("/rm/rmaccounts/" + STUB_CUSTOMER_ID + "/titles.*"), true))
+        .willReturn(new ResponseDefinitionBuilder()
+          .withStatus(500)));
+
+    RestAssured.given()
+      .spec(spec).port(port)
+      .header(new Header(RestConstants.OKAPI_URL_HEADER, wiremockUrl))
+      .header(TENANT_HEADER)
+      .header(TOKEN_HEADER)
+      .when()
+      .get("eholdings/titles?filter[name]=news")
+      .then()
+      .statusCode(500);
+  }
+}
```

Comment:
Might also be helpful to have a mock of an rm api error response and show how it is being mapped in terms of mod-kb-ebsco error respponse